### DISCUSSION TITLE
Mail button in the footer leads on to a blank page, to solve it

### DISCUSSION THREAD

**Discussion by shravanibhor** [chakra-ui/chakra-ui#8614] [General]
### Description

To remove the mail button in the footer as it leads on to a blank page when clicked.

### Problem Statement/Justification

![Screenshot (565)](https://github.com/chakra-ui/chakra-ui/assets/121655392/5f3c724e-32f0-4e90-a0a5-625bb2cf568e)
The mail button shown in the SS should either be removed or a specific mail should be added to it.

### Proposed Solution or API

The solution can include a small message which asks the user to connect through other ways, or else directly remove the button as its a unnecessary addition.

**Comment by segunadebayo** [comment]
Closing. This is stale and we are not acting on it.
